src/components/data-management/AddMetadataButton.jsx

```diff
@@ -3,12 +3,16 @@ import {
   Menu, Dropdown, Button,
 } from 'antd';
 import PropTypes from 'prop-types';
+import axios from 'axios';
 
 import { useSelector, useDispatch } from 'react-redux';
 
 import uploadMetadataFile from 'redux/actions/experiments/uploadMetadataFile';
 import { sampleTech } from 'utils/constants';
+import createCellLevelMetadata from 'redux/actions/experiments/createCellLevelMetadata';
+import { putPartInS3 } from 'utils/upload/processMultipartUpload';
```

Comment:
Why not use the multipart upload instead? That way we don't have to worry about the size

Reply:
Or is it more complicated?

Reply:
I started doing it with multipart, but asked @gerbeldo  about the size and he said "not more than a couple of dozen megabytes" so I changed it to this since it won't be needed for a small file like that.

Reply:
apart from the size limits, multipart tends to have better throughput so unless there is any disadvantage to using the multipart I'm not seeing, I think we should use it